docs/user/bots.rst

```diff
@@ -1243,6 +1243,38 @@ HTML Table Parser
  * `"type"`: set the `classification.type` statically, optional
  * `"html_parser"`: The HTML parser to use, by default "html.parser", can also be e.g. "lxml", have a look at https://www.crummy.com/software/BeautifulSoup/bs4/doc/
 
+JSON Custom Parser
+^^^^^^^^^^^^^^^^^
+
+**Configuration parameters**
+
+* `"json_data_format"`: Boolean, if list of data is within key of json object, optional. Default: false.
+* `"json_data_key"`: Key of json object where data list is present. string should be flatten_key, optional. To be used in conjunction with `"json_data_format"`. Default: `""`.
+   E.g.
+
+  .. code-block:: json
+
+     "json_data_format": true,
+     "json_data_key": "data.ipdata"
+
+  With above configuration, list of dict will be created from list present in json["data"]["ipdata"]. Each dict will then create atleast an event.
+* `"splitlines"`: Boolean, spit multiline data into list, optional. Default: `"false"`. Either `"json_data_format"` or `"splitlines"` can be used.`
```

Comment:
Please expand the explanation, examples are also helpful.

Reply:
Again in above json, if we have config `json_data_key`: _data_ and `json_data_format `: _True_, it will create two event with list of data present in json["data"] with proper harmonized field as per given in `translate_field` parameter.

Reply:
Why are `json_data_format` and `splitlines` mutually exclusive?

Btw, I think there's no test that covers actual line splitting (the tests with `splitlines=true` only have one line)